src/NuGet.Clients/NuGet.CommandLine/CommandManager.cs

```diff
@@ -50,7 +50,7 @@ public ICommand GetCommand(string commandName)
             return matchedCommand;
         }
 
-        public IDictionary<OptionAttribute, PropertyInfo> GetCommandOptions(ICommand command)
+        public virtual IDictionary<OptionAttribute, PropertyInfo> GetCommandOptions(ICommand command)
```

Comment:
We don't need to make this virtual if we mock the ICommandManager instead of the concrete CommandManager class in the tests.